src/feedback/activityIndicator/activityIndicator.jsx

```diff
@@ -159,19 +189,34 @@ class ActivityIndicator extends React.PureComponent {
             id,
             size,
             style,
+            theme,
         } = this.props;
-        const containerClasses = ClassNames('ui', BLOCK_CLASS, className);
+
+        const {
+            palette: {
+                cyan,
+                grey,
+            },
+        } = theme;
+
+        const rootClasses = ClassNames(
+            'ui',
+            BLOCK_CLASS,
+            className,
+        );
+
         const transformScaleDecimal = (1 / 68) * size;
+
         let backgroundColorStyle;
 
         switch (color) {
             case 'backgroundColorStatic':
-                backgroundColorStyle = colorStyles.backgroundColorStatic;
+                backgroundColorStyle = grey[400];
```

Comment:
Didn't we need the ESLINT destructuring suppression comments here...?  If not, no biggie for me.